okio/src/commonMain/kotlin/okio/SegmentPool.kt

```diff
@@ -15,10 +15,13 @@
  */
 package okio
 
+import kotlin.native.concurrent.ThreadLocal
+
 /**
  * A collection of unused segments, necessary to avoid GC churn and zero-fill.
  * This pool is a thread-safe static singleton.
  */
+@ThreadLocal
```

Comment:
Required to avoid a bunch of errors in Kotlin/Native unit tests. If this isn't desired I will look into alternatives.

Reply:
clever fix

Reply:
Tempted to make it use actual ThreadLocals on JVM